webapp/channels/src/components/suggestion/switch_channel_provider.tsx

```diff
@@ -256,23 +256,12 @@ const SwitchChannelSuggestion = React.forwardRef<HTMLLIElement, Props>((props, r
     }
     const showSlug = (isPartOfOnlyOneTeam || channel.type === Constants.DM_CHANNEL) && channel.type !== Constants.THREADS;
 
-    const getId = () => {
-        if (channel.type === Constants.DM_CHANNEL) {
-            if (prefix) {
-                return `quickSwitchInput_${(channel as FakeDirectChannel).userId}`;
-            }
-        }
-        return `quickSwitchInput_${channel.id}`;
-    };
-
     return (
         <SuggestionContainer
             ref={ref}
             data-testid={channel.name}
-            role='option'
             aria-labelledby={`${name.toLowerCase().replaceAll(' ', '-')}-item-name`}
             {...props}
-            id={getId()}
```

Comment:
Like in `AtMentionSuggestion` the `SuggestionList` sets the ID on the outer `li` element now